base/permuteddimsarray.jl

```diff
@@ -87,13 +87,68 @@ end
 
 """
     permutedims(A::AbstractArray, perm)
+    permutedims(A::AbstractMatrix)
 
-Permute the dimensions of array `A`. `perm` is a vector or a tuple of length `ndims(A)`
-specifying the permutation.
+Permute the dimensions (axes) of array `A`. `perm` is a tuple or vector of length `ndims(A)`
+integers specifying the permutation.
```

Comment:
"containing" is not quite right here, because it could imply that the length should be at *least* `ndims(A)`.  Alternatively, if you think of it in the sense of set containment, then `[1,1,1,1,1]` "contains" only one integer. 

I will change it to "of".